crates/win-api-wrappers/src/handle.rs

```diff
@@ -58,6 +58,18 @@ impl Handle {
         // SAFETY: Same preconditions as the called function.
         unsafe { Self::new(handle, true) }
     }
+
+    /// Wraps a pseudo Windows [`HANDLE`].
+    ///
+    /// # Safety
+    ///
+    /// - The caller should ensure that `handle` is a pseudo handle, as its validity is not checked.
+    pub unsafe fn new_pseudo_handle(handle: HANDLE) -> Self {
```

Comment:
@CBenoit FYI I was pulling my hair while trying to find why the process didn't want to start after my last refactoring, but it turned out that `current_process` function has been broken after https://github.com/Devolutions/devolutions-gateway/issues/998 was merged :( I added the separate `new_pseudo_handle` function to create handle without `invalid_handle` check (`CurrentProcess` returns `(HANDLE)-1` value)